docs/core/deploying/single-file.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Single file application
+description: Learn what single file application is and why you should consider using this application deployment model.
+author: lakshanf
+ms.author: lakshanf
+ms.date: 08/28/2020
+---
+# Single file deployment and executable
+
+Bundling all application dependent files into a single binary provides an application developer with the attractive option to deploy and distribute the application as a single file. This deployment model has been available since .NET Core 3.0 and has been enhanced in .NET 5.0 where the code is directly executed from the single file in the client machine. On .NET Core 3.0 version, all files were first self-extracted into a temporary location before running. On Windows .NET 5.0, single-file option will not bundle native binaries since a good debugging experience requires these not to be included in the single file currently. Linux will have all files bundled in .NET 5.0.
+
+Single File deployment is available for both [framework-dependent deployment model](index.md#publish-framework-dependent) and [self-contained application](index.md#publish-self-contained) although we anticipate that most developers will use Single file with client-side [self-contained application](index.md#publish-self-contained) managed applications for .NET Core 5.0. The single file deployment option can be combined with [Trim](trim-self-contained.md) and [ReadyToRun](../tools/dotnet-publish.md) publish options.
+
+There are some caveats that you need to be aware for single-file use, chief of which is the use path information to locate a file relative to the location of your application. Assembly. Location API will return an empty string which is the default behavior for assemblies loaded from memory. The compiler will give a warning for this API during build time to alert the developer to the specific behavior. If the path to the application directory is needed, AppContext.BaseDirectory, API will return the directory where the AppHost (the single-file bundle itself) resides. Managed C++ applications are not best suited for single-file deployment and we encourage application to be rewritten in C# or other managed language to be single-file compatible.
```

Comment:
Something got garbled in the first part of this paragraph -- "...which is the use path information..." isn't clear and "Assembly." stands by itself as if it were a sentence.  Should this be "use of path information" and "The Assembly.Location API"? Would be ideal to make Assembly.Location an xref link.

Same comment for "If the path..." sentence -- add "the" before "AppContext.BaseDirectory API (remove comma before API) and make it an xref link.

"not best suited" --> "not well suited"
"we encourage application to be rewritten in C#" --> "we recommend that you write applications in C#"

Reply:
Thanks, added these